app/models/ticket.py

```diff
@@ -25,16 +26,18 @@ class Ticket(db.Model):
 
     def __init__(self,
                  name,
-                 quantity,
-                 event,
+                 type,
```

Comment:
This won't cause any issue. Builtin function "type" will not conflict with the "type" attribute here. This "type" field will be used by referencing it like `ticket_obj.type`. We are already using another "type" field in the Events model.